scripts/exglobal_extractvars.sh

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env bash                                                                                                                                                                    
+################################################################################
+#   Script:    
+#
+source "${USHgfs}/preamble.sh"
+
+# Programs used
+export WGRIB2=${WGRIB2:-${wgrib2_ROOT}/bin/wgrib2}
+
+# Scripts used
+EXTRCTVARA=${EXTRCTVARA:-"${USHgfs}/atmos_extractvars.sh"}
+EXTRCTVARO=${EXTRCTVARO:-"${USHgfs}/ocnice_extractvars.sh"}
+EXTRCTVARW=${EXTRCTVARW:-"${USHgfs}/wav_extractvars.sh"}
+
+#Define a job-specific variable for FHMAX_HF_GFS
+if (( FHMAX_GFS < FHMAX_HF_GFS )); then
+  export FHMAX_HF_EV=FHMAX_GFS
+else
+  export FHMAX_HF_EV=FHMAX_HF_GFS
+fi
+
+#Extract variables for atmosphere
+if [[ "${DO_ATM}" == "YES" ]];then
+  if [[ ! -d "${DATA}/mem${ENSMEM}_atmos" ]]; then 
+    mkdir -p "${DATA}/mem${ENSMEM}_atmos" 
+  fi
+  ${EXTRCTVARA} "${DATA}/mem${ENSMEM}_atmos"
```

Comment:
I was using `mem${ENSMEM}_atmos` before I added this task to the global-workflow. Now that there is a rocoto task for each member, I agre there is no need to tag the working subdirectories with the member. Will remove "mem_${ENSMEM}_" from the name of the subdirectory for each component. Will also remove the mkdir portion here since this is already done in the ush scripts.